react/components/QuantitySelector.tsx

```diff
@@ -107,41 +168,90 @@ const QuantitySelector: FunctionComponent<Props & InjectedIntlProps> = ({
     onChange(validatedValue)
   }
 
-  const handleInputChange = (value: string) => {
-    const displayValue = validateDisplayValue(value, maxValue)
```

Comment:
Why remove `validateDisplayValue`? Was it breaking something or just redundant?

Reply:
it breaks stuff, since now we allow the use of non-numeric values in the input (periods and commas) this function would remove them as soon as we type, making it impossible to type a fraction :-/